src/rendering/renderable.cpp

```diff
@@ -292,7 +308,11 @@ void Renderable::registerUpdateRenderBinFromOpacity() {
 }
 
 float Renderable::opacity() const {
-    return _opacity * _fade;
+    // Rendering should depend on if camera is in the atmosphere and if camera is at the 
+    // dark part of the globe
+    float dimming = global::navigationHandler->camera()->atmosphereDimming();
+    dimming = _dimInAtmosphere ? dimming : 1.0;
```

Comment:
could be:
```cpp
float dimming = _dimInAtmosphere ? global::navigationHandler->camera()->atmosphereDimming() : 1.f;
```

that way we don't have to call atmosphereDimming for most items at all.
Alternatively:
```cpp
return _dimInAtmosphere ?
    _opacity * _fade * global::navigationHandler->camera()->atmosphereDimming() :
    _opacity * _fade;
```